ceph-dashboard-pull-requests-backend/config/definitions/ceph-dashboard-pull-requests-backend.yml

```diff
@@ -0,0 +1,63 @@
+- job:
+    name: ceph-dashboard-pull-requests-backend
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: arm64
+    display-name: 'ceph: dashboard Pull Requests backend'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph/
+      - rebuild:
+          auto-rebuild: true
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          allow-whitelist-orgs-as-admins: true
+          org-list:
+            - ceph
+          black-list-target-branches:
+            - luminous
+            - mimic
+          trigger-phrase: 'jenkins test dashboard backend'
+          skip-build-phrase: '^jenkins do not test.*'
+          only-trigger-phrase: true
+          github-hooks: true
+          permit-all: true
+          auto-close-on-fail: false
+          status-context: "ceph dashboard backend API tests"
+          started-status: "running ceph dashboard backend API tests"
+          success-status: "ceph dashboard backend API tests succeeded"
+          failure-status: "ceph dashboard backend API tests failed"
+
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph.git
+          branches:
+            - sha1
+          refspec: +refs/pull/*:refs/remotes/origin/pr/*
+          browser: auto
+          timeout: 20
+          skip-tag: true
+          wipe-workspace: true
+
+    builders:
+      - shell:
+          !include-raw:
+            - ../../../scripts/build_utils.sh
+            - ../../build/build
+      - shell: "cd src/pybind/mgr/dashboard; timeout 7200 ./run-backend-api-tests.sh"
```

Comment:
since we only fails when executing
```sh
pip install -r $CURR_DIR/requirements.txt
```

can we just drop the first two commands?

Reply:
Done!